Rewrite the Chart - Points Aggregation demo description(T929184) (#129)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md

> The **Chart** widget can aggregate data represented by series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 

Reviewer: ```suggestion
The **Chart** widget can aggregate series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
```

> In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The wider you set the selection range, the higher aggregation degree the control uses, and vice versa.

Reviewer 2: In the demo, are you setting the "selection" to a wide range? For example, do you mean the following:

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. If you set the selection to a wide range, the control will use a higher aggregation degree, and vice versa.
```

> In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.

Reviewer 3: ```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The more the width of its selection range is, the bigger the graph scale and the degree of aggregation are, and vice versa. 
```

> In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.

Reviewer 4: Just tightening up the other suggestion - it sound a bit better to me. 

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The wider you set the selection range, the higher aggregation degree the control uses, and vice versa.
```

> Aggregation on zoom is one of many possible use cases for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other scenarios.

Reviewer 2: ```suggestion
Aggregation on zoom is one of many possible use cases for this feature. For information on how to configure aggregation for other scenarios, refer to the following description: [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/).
```

> Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other use cases.

Reviewer 3: ```suggestion
Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description on how to configure aggregation for other use cases.
```

> Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other use cases.

Reviewer 4: I didn't want to repeat "cases" in two sentences. "One of many possible" sounds a bit better to me than "only one". The latter is somewhat vague - do we mean "one of many" or "the only one"?

```suggestion
Aggregation on zoom is one of many possible use cases for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other scenarios.
```

> In this demo, the chart's [visual range](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/visualRange/) depends on the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/)'s settings (see the [onValueChanged](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/Configuration/#onValueChanged) handler). When the selected range is reduced, the series automatically becomes less aggregated. To change this behavior and set the fixed aggregation degree, specify the [aggregationInterval](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/aggregationInterval/) option of an axis.

Reviewer: It is better to being from the beginning:

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.
```

I also removed the last sentence for two reasons: 1) `aggregationInterval` is not the only means of controlling the degree of aggregation - there's also `aggregationGroupWidth`; 2) this demo has an option called `aggregationInterval` set for the RangeSelector, and it is not the same option. I think it is better to send the user to the `aggregation` description and let him find out the details from there. Hence, the suggestion for the paragraph below.

> Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) option description to see possible options for different series types.

Reviewer: ```suggestion
Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other use cases.
```
